packages/fast-components-react-base/src/listbox-item/listbox-item.tsx

```diff
@@ -125,10 +133,9 @@ class ListboxItem extends Foundation<
      * Handle the keydown event of the item
      */
     private handleClick = (e: React.MouseEvent<HTMLDivElement>): void => {
-        if (this.props.disabled) {
+        if (this.props.disabled || e.defaultPrevented) {
```

Comment:
Because we provide a render function for the display component it is possible a developer could do something we don't expect there, and should there be interactions there a developer could preventDefault on clicks and key presses they want the parent control to ignore.  Same thing with the children of the list.

Reply:
Modified click and keydown functions to not invoke if default prevented, but otherwise calls user provided functions (ie. they can check for default prevented themselves)